snippets/functions/boilerplate/manifest.json

```diff
@@ -0,0 +1,31 @@
+{
+  "category": "Boilerplate Template by Function Type",
+  "categoryId": "4756072f-b3c7-4469-b038-cdf6b346789c",
+  "viewType": "functionSnippet",
+  "snippets": [
+    {
+      "id": "7c84ed29-b8b2-466c-af3d-c240889d3112",
+      "title": "Database Trigger",
+      "snippet": "/databaseTrigger.js",
+      "description": "Example shows a template for a DB trigger function, which always accepts a changeEvent in the header."
```

Comment:
In the header? Might be my lack of knowledge but I assume you're referring to the function signature that expects to receive a change event?

We can as "as argument" to be consistent with other descriptions we have in this file.

Reply:
Ok, ill change it to `as argument` instead of `in the header` this was the proposed description that Laura gave.